spread.yaml

```diff
@@ -276,36 +276,34 @@ suites:
    summary: core24 specific tests
    systems:
      - ubuntu-24.04*
+   environment:
+     SNAPCRAFT_BUILD_ENVIRONMENT: ""
+   prepare: |
+     # Workaround to be able to test the snaps
+     if snap install core24; then
+       echo Remove this workaround
+       exit 1
+     else
+       snap install core24 --edge
+     fi
 
- tests/spread/core24/environment/:
+
+ tests/spread/core24-suites/environment/:
    summary: core24 environment tests
    systems:
      - ubuntu-22.04*
```

Comment:
A bunch of these tests run in destructive mode, are they supposed to run on jammy still?

Reply:
good catch! I thought I did that update

Reply:
ok, with the current setup:

```
2024-03-05T14:38:09.8946667Z     - google:ubuntu-22.04-64:tests/spread/core24-suites/environment/test-variables:test_variables
2024-03-05T14:38:09.8949593Z     - google:ubuntu-22.04-64:tests/spread/core24-suites/manifest/manifest-creation:envvars
2024-03-05T14:38:09.8953244Z     - google:ubuntu-22.04-64:tests/spread/core24-suites/patchelf/classic-patchelf
2024-03-05T14:38:09.8970397Z     - google:ubuntu-22.04-64:tests/spread/core24-suites/patchelf/strict-patchelf
```

With Ubuntu 24.04 -> https://github.com/canonical/snapcraft/pull/4634/commits/39541c66148d30fc9a8f8d9bb6641639267f1e90

```
    - google:ubuntu-24.04-64:tests/spread/core24-suites/environment/paths:staged_common_library
    - google:ubuntu-24.04-64:tests/spread/core24-suites/environment/test-variables:test_variables
    - google:ubuntu-24.04-64:tests/spread/core24-suites/manifest/manifest-creation:envvars
    - google:ubuntu-24.04-64:tests/spread/core24-suites/patchelf/classic-patchelf
    - google:ubuntu-24.04-64:tests/spread/core24-suites/patchelf/strict-patchelf
    - google:ubuntu-24.04-64:tests/spread/core24-suites/scriptlets/scriptlet-failures
```

The last one is probably my old version of spread, the patchelf ones I think I can fix up to the point where they properly fail again due to our lack of classic confinement support -> https://github.com/canonical/snapcraft/pull/4634/commits/e8f688eb0e9a3061833e39d306f53fd96d33ea7c